front-end/page_target/SnvByGene_Config.json

```diff
@@ -118,12 +122,14 @@
   {
     "id": "pedcbioPedotOncoprintPlotURL",
     "label": "PedcBio PedOT oncoprint plot",
+    "exportLabel": "pedcbioPedotOncoprintPlot",
```

Comment:
```suggestion
    "exportLabel": "pedcbioPedOTOncoprintPlot",
```

Reply:
Thank you for the Suggestions. Since the exportLabel value will all go throw a camel case formatter, in this case pedcbioPed**OT**OncoprintPlot will became pedcbioPed**Ot**OncoprintPlot. I hope that is still acceptable.

Reply:
@Nahomtes Thank you for reply. "pedcbioPedOtOncoprintPlot" should be OK. These suggestions intend to treat "PedOT" the same way as "oncoKB". I did not notice that the "oncoKB" `exportLabel` is "oncoKb".

Reply:
Good. I will update these changes.